extensions/amp-sticky-ad/1.0/test/test-amp-sticky-ad.js

```diff
@@ -157,20 +158,42 @@ describes.realWin('amp-sticky-ad 1.0 version', {
       expect(impl.element.children[2].tagName).to.equal('BUTTON');
     });
 
-    it('should listen to amp:built, amp:load:end', () => {
-      impl.ad_.isBuilt = () => {
-        return false;
+    it('should wait for built and load-end signals', () => {
+      impl.ad_.isBuilt = () => false;
+      impl.vsync_.mutate = function(callback) {
```

Comment:
one last question. I see we use `vsync_.mutatePromise` but mock `vsync_mutate` here. What are the differences?